encrypted-media-respec.html

```diff
@@ -3209,10 +3209,46 @@
         </section>
       </section>
         
+      <section id="exposed-value-requirements">
+        <h3>Values Exposed to the Application</h3>
+        <p>Values exposed to the application could be used to identify the client or user, regardless of whether they are designed to be identifiers.
+          This section defines requirements for avoiding or at least mitigating such concerns.
+          There are additional requirements for <a href="#identifier-requirements">Identifiers</a>.
+        </p>
+
+        <section id="per-origin-per-profile-values">
+          <h3>Use Per-Origin Per-Profile Values</h3>
+          <!-- Issue #101 may affect this text. -->
+          <p>
+            All <a href="#distinctive-value">distinctive values</a> exposed to or inferable by the application MUST be unique per <a def-id="origin"></a> and <a def-id="browsing-profile"></a>.
+            That is, the value(s) used for one <a def-id="origin"></a> using the APIs defined in this specification MUST be different from those used for any other origin using the APIs,
+            and value(s) used in one <a def-id="browsing-profile"></a> MUST be different from those used for any other profile, regardless of origin.
+          </p>
```

Comment:
How does this apply to session ids ? Presumably if you pick session ids at random, then you could end up using the same one for different sessions in different origins.